src/arch/InputHandler/InputHandler_DirectInput.cpp

```diff
@@ -339,10 +329,7 @@ InputHandler_DInput::UpdatePolled(
 				return;
 
 			if (hr != DI_OK) {
-				LOG->MapLog(
-				  "UpdatePolled",
-				  hr_ssprintf(hr, "Failures on polled keyboard update")
-					.c_str());
+                Locator::getLogger()->trace(hr_ssprintf(hr, "Failures on polled keyboard update"));
```

Comment:
We want to be careful converting MapLogs to traces due to their possibly high traffic usage. 
In this specific case, it might be better to make this a warn or a debug level message. If it crops up as an issue later, we could remove this log entirely or find a new solution.

Reply:
Changed to debug.